src/components/Collapse.tsx

```diff
@@ -56,6 +56,9 @@ type Props = {
   onExpand?: () => void
   className?: string
   arrowComponent?: ReactNode
+  headerStyle?: CSSProperties
+  headerBorderRadius?: string
+  arrowStyle?: CSSProperties
 }
 
 export const CollapseItem: React.FC<Props> = ({
```

Comment:
I suggest add toggle animation for this component as we have good UI here but quite a bad visual.
Also I see the Header color of these toggle are different than design

Reply:
nice catch